src/configuration/sales/braintree.md

```diff
@@ -2,91 +2,209 @@
+|Enable Vault for Card Payments|Website|When enabled, provides secure storage for customer payment information, so customers don't have to reenter their credit card information for each purchase. Options: Yes / No.|
+|Enable Vault CVV Re-verification|Website|When enabled, Validation will be done for the CVV rules setup under Braintree Account. Options: Yes / No.|
+
 
 ## Advanced Braintree Settings
 
-![]({% link images/images/config-sales-payment-methods-braintree-advanced.png %}){: .zoom}
+![]({% link images/images/config-sales-payment-methods-braintree-advance-configuration.png %}){: .zoom}
 _Braintree Advanced Settings_
 
 |Field|[Scope]({% link configuration/scope.md %})|Description|
 |--- |--- |--- |
 |Vault Title|Website|A descriptive title for your reference that identifies the vault where your customer card information is stored.|
 |Merchant Account ID|Website|The Merchant ID that is to be associated with Braintree transactions from this website. If left blank, the default merchant account from your Braintree account is used.|
-|Advanced Fraud Protection|Website|Determines if Braintree’s Advanced Fraud Protection is applied to transactions. Options: Yes / No <br/><br/>**Your Kount ID** - When Advanced Fraud Protection is enabled, the system requires your Kount ID to integrate with the fraud protection platform. If necessary, [contact Braintree](mailto:accounts@braintreepayments.com?subject=Kount Account for Advanced Fraud Protection) to set up your [Kount](https://www.kount.com/) account. <br/><br/>**_Note:_** Advanced Fraud Protection must also be enabled in your Braintree account.|
+|Advanced Fraud Protection|Website|Determines if Braintree’s Advanced Fraud Protection is applied to transactions. Options: Yes / No |
 |Debug|Website|Determines if communications between the Braintree system and your store are recorded in a log file. Options: Yes / No|
 |CVV Verification|Website|Determines if customers are required to provide the three-digit security code from the back of a credit card. Options: Yes / No|
 |Credit Card Types|Website|Specifies each credit card that you accept as payment through  Braintree. Press and hold `Ctrl` (or `Command` on Mac) to select a combination of cards.|
 |Sort Order|Website|Determines the order that Braintree is listed with other payment methods during checkout.|
 
+
+
+### Kount
+![]({% link images/images/config-sales-payment-methods-braintree-kount-configuration.png %}){: .zoom}
+_Kount Configuration_
+
+
+|Kount Configuration| | When Advanced Fraud Protection option will be enabled, Kount configuration option will appear|
+|Bypass Fraud Protection Threshold|Website|Advanced fraud protection checks will be bypassed if this threshold is met or exceeded. Leaving this field blank will disable this option.|
+|ENS URL|Website|Under Kount AWC control panel, ENS URL needs to be added for you|
```

Comment:
I'm confused as to what this means. Do we mean that they need to enter the URL that is in the AWC control panel for their Kount account?

```suggestion
|ENS URL|Website|Under Kount AWC control panel, ENS URL needs to be added for you|
```